### ISSUE TITLE
eslint-disable directives require C-style comments

### ISSUE THREAD

**Issue by danvk** [typescript-eslint/tslint-to-eslint-config#645]
### 🐛 Bug Report

-   `tslint-to-eslint-config` version: 1.0.0
-   ESLint version: v7.5.0
-   Node version: v12.16.3

#### Actual Behavior

I had some code like this:

```ts
// See https://github.com/mikolalysenko/hash-int
export function hashInt(x: number) {
  // tslint:disable:no-bitwise
  const HASH = new Uint32Array(1);
  HASH[0] = x | 0;
  HASH[0] -= HASH[0] << 6;
  HASH[0] ^= HASH[0] >>> 17;
  HASH[0] -= HASH[0] << 9;
  HASH[0] ^= HASH[0] << 4;
  HASH[0] -= HASH[0] << 3;
  HASH[0] ^= HASH[0] << 10;
  HASH[0] ^= HASH[0] >>> 15;
  return HASH[0];
  // tslint:enable
}
```

When I migrate with `--comments`, this becomes:

```ts
export function hashInt(x: number) {
  // eslint-disable no-bitwise
  const HASH = new Uint32Array(1);
  HASH[0] = x | 0;
  HASH[0] -= HASH[0] << 6;
  HASH[0] ^= HASH[0] >>> 17;
  HASH[0] -= HASH[0] << 9;
  HASH[0] ^= HASH[0] << 4;
  HASH[0] -= HASH[0] << 3;
  HASH[0] ^= HASH[0] << 10;
  HASH[0] ^= HASH[0] >>> 15;
  return HASH[0];
  // eslint-enable
}
```

Unfortunately this allows all the no-bitwise errors to surface in eslint.

#### Expected Behavior

It seems that eslint requires C-style comments for eslint-disable directives:

```ts
export function hashInt(x: number) {
  /* eslint-disable no-bitwise */
  const HASH = new Uint32Array(1);
  HASH[0] = x | 0;
  HASH[0] -= HASH[0] << 6;
  HASH[0] ^= HASH[0] >>> 17;
  HASH[0] -= HASH[0] << 9;
  HASH[0] ^= HASH[0] << 4;
  HASH[0] -= HASH[0] << 3;
  HASH[0] ^= HASH[0] << 10;
  HASH[0] ^= HASH[0] >>> 15;
  return HASH[0];
  /* eslint-enable */
}
```

Note that this _doesn't_ seem to be the case for `eslint-disable-line` or `eslint-disable-next-line`.

**Comment by KingDarBoja** [comment]
Hi @danvk, thanks for reporting this bug, you are correct as stated by the official [ESLint docs](https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments), those require using code blocks whereas the `eslint-disable-next-line` and `eslint-disable-line` uses line comments.

**Comment by KingDarBoja** [comment]
Hi again @danvk, after some checks and fixing our test suite, I noticed your `tslint` comments using `//` instead of `/*` as stated by [TSLint docs](https://palantir.github.io/tslint/usage/rule-flags/). Unfortunately, our logic requires the use of block comments as we use the TypeScript AST to check the comment kind (either Single or MultiLine) as you can see at https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/src/comments/parseFileComments.ts#L42

**Comment by danvk** [comment]
Hi @KingDarBoja. So what's the decision here? While it may be an accident that tslint allows `//`-style comments to disable rules for multiple lines, it's definitely existing behavior that would be nice to convert when migrating comments.

**Comment by KingDarBoja** [comment]
Let me try again if I can find a way to support that alternative on the linked PR.
